applications/pelion_client/README.rst

```diff
@@ -43,16 +46,20 @@ For more information about these functions, see the `Pelion's factory_configurat
 Using a valid certificate allows the device to successfully register (connect) to the Device Management.
 At this stage, the device is assigned a new device identity.
 
+.. warning::
```

Comment:
Do not use warnings unless there is something that puts the user's health or life at risk: https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Style+and+usage+guide

Instead, use a note.

Reply:
btw - @greg-fer , if you will be changing this part, can you use "button 1" instead of "any of the buttons". This will be valid after https://github.com/nrfconnect/sdk-nrf/pull/4705 is merged